sui_core/src/authority/authority_store.rs

```diff
@@ -307,6 +307,14 @@ impl<
             .collect())
     }
 
+    pub fn get_object_version(
```

Comment:
`get_object_by_key` is probably a better name

Reply:
done